docs/release-notes/index.rst

```diff
@@ -0,0 +1,259 @@
+.. _release-notes:
+
+Release notes
+=============
+
+This page lists past release notes for Starcraft, summarising new features, bug
+fixes and backwards-incompatible changes in each version. It also contains the
+release and support policies for Starcraft.
+
+
+Current releases
+----------------
+
+
+<latest release>
+~~~~~~~~~~~~~~~~
+
+- <link to latest release, update, such as 8.2.0>
+- <link to latest release, update, such as 8.1.0>
+- <link to latest release, initial, such as 8.0.0>
+
+
+<parallel release>
+~~~~~~~~~~~~~~~~~~
+
+<If necessary, add guidance and caveats about these older releases, such as
+"Snapcraft 7 is available for building core18 snaps. When building for a newer
+base, use Snapcraft 8.">
+
+- <link to parallel release, update, such as 7.2.0>
+- <link to parallel release, update, such as 7.1.0>
+- <link to parallel release, initial, such as 7.0.0>
+
+
+Past releases
+-------------
+
+
+<past release>
+~~~~~~~~~~~~~~
+
+- <link to past release, update, such as 6.2.0>
+- <link to past release, update, such as 6.1.0>
+- <link to past release, initial, such as 6.0.0>
+
+
+.. _release_policy_and_schedule:
+
+Release policy and schedule
+---------------------------
+
+Canonical is committed to supporting the <"latest major release" or "last two
+major releases"> of Starcraft. <Optional: "We forward-port changes in older
+releases to the latest release, if they're compatible.">
+
+Starcraft is released when it achieves development milestones in its product
+lifecycle. It doesn't follow a predefined release cadence.
+
+Starcraft release naming follows the Semantic Versioning 2.0.0 scheme with
+numbers for major, minor, and patch versions.
+
+.. list-table::
+    :header-rows: 1
+
+    * - Version
+      - Example
+      - Significance
+    * - Major
+      - **3**.1.2
+      - <Apps: "A change that drops support for an earlier software base.">
```

Comment:
@lengau Is this and lines 89-90 acceptable for Charmcraft?

Reply:
Looks fine to me.